definitions/features/satellite.rb

```diff
@@ -0,0 +1,25 @@
+class Features::Satellite < ForemanMaintain::Feature
+  include ForemanMaintain::Concerns::Downstream
+
+  metadata do
+    label :satellite
+
+    confine do
+      feature(:package_manager).satellite_installed?
+    end
+  end
+
+  def current_version
+    @current_version ||= rpm_version(package_name) || version_from_source
+  end
+
+  private
+
+  def package_name
+    feature(:package_manager).satellite_package
```

Comment:
The satellite feature should be primary source for the satellite package name. Not the package manager.